frontend/src/components/Staffing/AddEngagementForm.tsx

```diff
@@ -151,6 +159,7 @@ export function AddEngagementForm({
         }}
         className="flex flex-col gap-4 items-center"
       >
+        <div className="error-message">{errorMessage}</div>
```

Comment:
```suggestion
        {errorMessage && <div className="text-red italic">{errorMessage}</div>}
```
Det kan være lurt å ikke tegne opp div-en med mindre du trenger den. 

Også styles komponenter via tailwind, så du vil heller skrive `className="text-red italic"`

globals.css blir veldig lett ganske stor om man har klasser for alle komponenter der sjø. I samme slengen er det lurt å slette klassen du laget der